client/src/api/Card.js

```diff
@@ -1,4 +1,16 @@
 import axios from 'axios';
+import ApiResultFactory from './ApiResult';
+
+const defaultErrorHandler = (error) => {
+  // Check if error is axios error
+  if (error.response) {
+    const {errors} = error.response.data;
+    return ApiResultFactory.withErrors(errors);
+  }
+
+  console.log(error.message);
+  return {};
```

Comment:
Are these 2 lines mostly to avoid eslint errors? I feel they aren't necessary for an error handler

Reply:
Ideally those lines would be where non-XHR errors are handled (`if(error.response)` is a hacky way of checking "oh did axios throw this?"). For example, earlier in development, there was an error with destructuring a null request body. For a default error handler, I felt logging the error was good enough.